pupa/utils/model_ops.py

```diff
@@ -0,0 +1,113 @@
+from django.db import transaction
+from django.db.models import get_models, Model
+from django.contrib.contenttypes.generic import GenericForeignKey
+
+import json
+import logging
+
+logger = logging.getLogger("")
+
+@transaction.commit_on_success
+def merge_model_objects(primary_object, alias_objects=[], keep_old=False):
```

Comment:
I replied on the other PR, but i'll repeat here. I think we should go ahead with @rshorey's merge code. I'm suggesting this for two reasons:

- this got the job done for the particular usecase that I had, but that usecase isn't going to exist once we move to the kafka-based record linkage approach
- it makes all the sense in the world for the merge operations to exist in python-opencivicdata-django